.eslintrc

```diff
@@ -2,7 +2,6 @@
   "parser": "babel-eslint",
   "rules": {
     "quotes": [ 2, "single" ],
-    "linebreak-style": [ 2, "unix" ],
```

Comment:
I don't care for that rule as long as you configure your git correctly for line endings:
https://help.github.com/articles/dealing-with-line-endings/#platform-windows